docs/SECURE_PLUGIN_COMMUNICATION.md

```diff
@@ -0,0 +1,124 @@
+<!--
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2017 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Secure Plugin Communication
+
+Snap communicates with plugins over gRPC protocol, which in general transfers data in plaintext.
+Snap allows securing communication with plugins by opening TLS channels and using certificates to authenticate plugins and framework.
+
+## Usage 
+
+This walkthrough assumes you have downloaded a Snap release as described in [Getting Started](../README.md#getting-started).
+
+### Shortest guide
+
+Assuming all the test files are available, the following steps will result in secure plugin communication:
```

Comment:
Added reference to test procedure, which makes it clear.
Fixed [here](https://github.com/intelsdi-x/snap/pull/1591/commits/f2bae041d5adce4aed85d5e388ac4464468c51e7#diff-02719769e9c631d8b2070352a588d15eR51)